src/main/java/me/crespel/runtastic/model/User.java

```diff
@@ -0,0 +1,56 @@
+package me.crespel.runtastic.model;
+
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonFormat.Shape;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+import lombok.Data;
+
+/**
+ * Runtastic model for user (\\User\\user.json).
+ * @author Christian IMFELD (imfeldc@gmail.com)
+ */
+@Data
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class User {
+    
+    private String firstName;
+    private String lastName;
+	@JsonFormat(shape = Shape.STRING, pattern = "yyyy-MM-dd HH:mm:ss Z")
+	private Date createdAt;
+    private String beginWeek;
+	@JsonFormat(shape = Shape.STRING, pattern = "yyyy-MM-dd")
+	private Date birthday;
+    private String cityName;
+	@JsonFormat(shape = Shape.STRING, pattern = "yyyy-MM-dd HH:mm:ss Z")
+	private Date confirmedAt;
+    private String currentSignInIp;
+    private String email;
+    private String fbProxiedEMail;
+    private String fbUserId;
+    private Boolean fitbitConnected;
+    private String formatOfDate;
+    private String formatOfTime;
+    private String gender;
+    private Double height;
```

Comment:
Double values can sometimes have precision issues, consider using BigDecimal instead.

Reply:
Thanks! Double replaced with BigDecimal. Will be part of next dleivery.